packages/block-library/src/classic.scss

```diff
@@ -14,6 +14,10 @@
 	font-size: 1.125em;
 }
 
+.wp-block-file {
+	margin-bottom: 1.5em;
+}
+
```

Comment:
As noted in the `__experimentalStyle` thread, we'll need to remove this style from `classic.scss` to maintain the lower specificity default set in `style.scss` required by layout supports.